api/.env.dev

```diff
@@ -10,6 +10,13 @@ GH_CLIENT_ID=""
 GH_CLIENT_SECRET=""
 GHE_URL=""
 
+GL_CLIENT_ID=""
```

Comment:
does this pr ? enable GL an  GLE to ?
if yes then should be part of pr desc/heading